### ISSUE TITLE
porkbun DNSPLUGIN seems to be broken or missing after latest update

### ISSUE THREAD

**Issue by singularity098** [linuxserver/docker-swag#325]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

After recent update of my swag container, I am seeing the following error:

`Please set the DNSPLUGIN variable to a valid plugin name. See docker info for more details.`

My DNSPLUGIN = porkbun

This worked and I could successfully generate certificates for months prior to the most recent container update.

To narrow it down to porkbun, I tried setting DNSPLUGIN = cloudflare, and also tried set to google.  Both made attempts to generate the certificate, unlike porkbun which now doesn't recognize porkbun as a valid value and gives up.  It's as if some porkbun related code disappeared.

### Expected Behavior

Expected that porkbun DNSPLUGIN works and generates the certificates.

### Steps To Reproduce

Launch docker with porkbun plugin.

### Environment

```markdown
- OS: Unraid
- How docker service was installed: Pre-installed on Unraid
```

### CPU architecture

x86-64

### Docker creation

```bash
docker run
  -d
  --name='swag-lan'
  --net='proxynet-lan'
  -e TZ="America/New_York"
  -e HOST_OS="Unraid"
  -e HOST_HOSTNAME="******"
  -e HOST_CONTAINERNAME="swag-lan"
  -e 'URL'='*****.***'
  -e 'VALIDATION'='dns'
  -e 'SUBDOMAINS'='****,*****,****'
  -e 'CERTPROVIDER'=''
  -e 'DNSPLUGIN'='porkbun'
  -e 'PROPAGATION'=''
  -e 'DUCKDNSTOKEN'=''
  -e 'EMAIL'='*********'
  -e 'ONLY_SUBDOMAINS'='true'
  -e 'EXTRA_DOMAINS'=''
  -e 'STAGING'='false'
  -e 'MAXMINDDB_LICENSE_KEY'=''
  -e 'PUID'='99'
  -e 'PGID'='100'
  -e 'UMASK'='022'
  -l net.unraid.docker.managed=dockerman
  -l net.unraid.docker.webui='http://[IP]:[PORT:80]'
  -l net.unraid.docker.icon='https://raw.githubusercontent.com/linuxserver/docker-templates/master/linuxserver.io/img/linuxserver-ls-logo.png'
  -p '443:443/tcp'
  -p '800:80/tcp'
  -v '/mnt/cache/appdata/swag-lan':'/config':'rw'
  --cap-add=NET_ADMIN 'lscr.io/linuxserver/swag'
```

### Container logs

```bash
text  error  warn  system  array  login  

[migrations] started
[migrations] 01-nginx-site-confs-default: skipped
[migrations] done

-------------------------------------
          _         ()
         | |  ___   _    __
         | | / __| | |  /  \
         | | \__ \ | | | () |
         |_| |___/ |_|  \__/


Brought to you by linuxserver.io
-------------------------------------

To support the app dev(s) visit:
Certbot: https://supporters.eff.org/donate/support-work-on-certbot

To support LSIO projects visit:
https://www.linuxserver.io/donate/
-------------------------------------
GID/UID
-------------------------------------

User uid:    99
User gid:    100
-------------------------------------

using keys found in /config/keys
Variables set:
PUID=99
PGID=100
TZ=America/New_York
URL=djent.me
SUBDOMAINS=*****,*****,****
EXTRA_DOMAINS=
ONLY_SUBDOMAINS=true
VALIDATION=dns
CERTPROVIDER=
DNSPLUGIN=porkbun
EMAIL=**********
STAGING=false

Please set the DNSPLUGIN variable to a valid plugin name. See docker info for more details.
```

**Comment by nemchik** [comment]
https://github.com/linuxserver/docker-swag#versions
https://github.com/linuxserver/docker-swag/releases/tag/2.2.0-ls174
Note the changelog listing BREAKING CHANGE

Unfortunately the porkbun plugin needs to be updated to support certbot 2.x. The dev seems to have already started the change, but not yet released a new version.
https://github.com/infinityofspace/certbot_dns_porkbun/issues/61

We updated swag to Alpine 3.17 with certbot 1.x and it broke armv7. Our options were to roll back three Alpine update or push an update to certbot 2.x which fixed armv7, but would require us to temporarily remove porkbun. We opted for the later.

Your best option for the moment is to downgrade to 1.32.0
https://github.com/linuxserver/docker-swag/pkgs/container/swag/64879474?tag=1.32.0
